packages/lerna/src/commands/add-caching/index.ts

```diff
@@ -74,13 +90,13 @@ class AddCachingCommand extends Command {
         message:
           "Which scripts are cacheable? (Produce the same output given the same input, e.g. build, test and lint usually are, serve and start are not.)\n",
         choices: this.uniqueScriptNames,
+        default: existingCacheableOperations,
```

Comment:
Same as above; this allows inquirer to pre-select all targets that have `"cache": true` already configured